src/main/java/com/sprint/mission/discodeit/service/basic/BasicChannelService.java

```diff
@@ -0,0 +1,147 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.dto.channel.ChannelDTO;
+import com.sprint.mission.discodeit.dto.channel.PrivateChannelCreateRequest;
+import com.sprint.mission.discodeit.dto.channel.PublicChannelCreateRequest;
+import com.sprint.mission.discodeit.dto.channel.PublicChannelUpdateRequest;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.ChannelType;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.entity.User;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.MessageRepository;
+import com.sprint.mission.discodeit.repository.ReadStatusRepository;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import com.sprint.mission.discodeit.service.ChannelService;
+import jakarta.transaction.Transactional;
+import java.time.Instant;
+import java.util.Comparator;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+
+@Service
+@RequiredArgsConstructor
+public class BasicChannelService implements ChannelService {
+
+  private final ChannelRepository channelRepository;
+  private final UserRepository userRepository;
+  private final ReadStatusRepository readStatusRepository;
+  private final MessageRepository messageRepository;
+
+  @Transactional
+  @Override
+  public ChannelDTO create(PublicChannelCreateRequest publicChannelCreateRequest) {
+    if (channelRepository.existsByName((publicChannelCreateRequest.channelName()))) {
+      throw new IllegalArgumentException("이미 존재하는 채널 이름입니다.");
+    }
+
+    Channel channel = new Channel(ChannelType.PUBLIC, publicChannelCreateRequest.channelName(),
+        publicChannelCreateRequest.description());
+    channelRepository.save(channel);
+    return toDto(channel);
+  }
+
+  @Transactional
+  @Override
+  public ChannelDTO create(PrivateChannelCreateRequest privateChannelCreateRequest) {
+    Channel channel = new Channel(ChannelType.PRIVATE, null, null);
+    Channel createdChannel = channelRepository.save(channel);
+
+    privateChannelCreateRequest.participantIds().forEach(userId -> {
+      User participant = userRepository.findById(userId)
+          .orElseThrow(() -> new NoSuchElementException("유저가 존재하지 않습니다."));
+      ReadStatus rs = new ReadStatus(createdChannel, participant);
```

Comment:
약어 사용은 나중에 코드를 이해하기 어렵게 만들기에 지양하는게 좋습니다.  
```suggestion
      ReadStatus readStatus = new ReadStatus(createdChannel, participant);
```